src/main/java/strategy/ReturnOperation.java

```
@@ -0,0 +1,13 @@
+package strategy;
+
+import core.basesyntax.FruitTransaction;
+import db.Storage;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void handle(FruitTransaction transaction) {
```

The task requires handling invalid input data, including negative quantities. You should add a check here to ensure `transaction.getQuantity()` is not negative. A customer cannot return a negative amount of fruit. If the quantity is negative, you should throw an exception.